src/components/pod/pod.component.tsx

```diff
@@ -39,9 +39,9 @@ export interface PodProps extends MarginProps {
   /** Prop to apply a theme to the Pod */
   variant?: PodVariant;
   /** Title for the pod h4 element always shown */
-  title?: string | React.ReactNode;
+  title?: React.ReactNode;
   /** Optional subtitle for the pod */
-  subtitle?: string;
+  subtitle?: React.ReactNode;
```

Comment:
suggestion: Potential side effect that something other than a JSX element or string is passed

React.ReactNode type will also allow unsupported types through like `number`. We could purely check for `string | React.ReactElement` or potentially allow them through and handle them in the code? We can discuss or see what the next reviewer suggests?